src/schema/v2/homeView/mixer/HomeViewMixer.ts

```diff
@@ -0,0 +1,37 @@
+import { HomeViewSection } from "schema/v2/homeView/sections"
+import { ResolverContext } from "types/graphql"
+import { SectionFilter } from "./SectionFilter"
+
+/**
+ * Orchestrates the application of multiple filters to a list of sections.
+ */
+export class HomeViewMixer {
```

Comment:
This all looks great design wise 👌 I just really wonder if it can all be achieved without any use of classes, instead leveraging simple functions without lingo. The JS world just doesn't intersect OOP much.

Reply:
(The last time I encountered an 'abstract class' in the wild was back in my ActionScript 3 days, about 15-20 years ago!)